nsv13/code/game/machinery/computer/helm.dm

```diff
@@ -9,6 +9,10 @@
 /obj/machinery/computer/ship/helm/syndicate
 	req_one_access = list(ACCESS_SYNDICATE)
 
+/obj/machinery/computer/ship/helm/Destroy()
+	linked?.helm -= src
```

Comment:
helm is just a object reference var, not a list var.
do something like if
(linked?.helm == src)
    linked.helm = null

Reply:
Coder brain setting in... thanks for catching that

Reply:
the fact that byond allows this... 😩

Reply:
The fact that byond allows this.. 😫